crates/matrix-sdk-crypto/src/store/memorystore.rs

```diff
@@ -54,6 +54,7 @@ pub struct MemoryStore {
     account: StdRwLock<Option<Account>>,
     sessions: SessionStore,
     inbound_group_sessions: GroupSessionStore,
+    outbound_group_sessions: StdRwLock<Vec<OutboundGroupSession>>,
```

Comment:
Wouldn't a `BTreeMap` from `RoomId` to `OutboundGroupSession` work better here? Not that it matters much.

Reply:
Thanks, done in 1e11ac406f